lib/themes/dosomething/paraneue_dosomething/scss/content/_campaign.scss

```diff
@@ -18,7 +18,9 @@
 
 .campaign--action .container--prove,
 .campaign--sms .container--do {
-  background: $purple asset-url("images/campaign-pattern-default.png") repeat center 63px;
+  // @TODO: See if this needs to be removed with v2 of Reportback Prove it section.
+  // background: $purple asset-url("images/campaign-pattern-default.png") repeat center 63px;
+  background-color: $yellow;
```

Comment:
Are we launching this yellow background color before Reportback v2? (Wondering if this should be hidden behind a feature flag.)